background.html

```diff
@@ -6,10 +6,14 @@
 
     <!-- include background scripts -->
     <script type="text/javascript" src="privly-applications/shared/javascripts/local_storage.js"></script>
+    <script type="text/javascript" src="privly-applications/shared/javascripts/storage.js"></script>
+    <script type="text/javascript" src="privly-applications/shared/javascripts/options.js"></script>
+    <script type="text/javascript" src="privly-applications/shared/javascripts/glyph.js"></script>
     <script type="text/javascript" src="javascripts/background_scripts/first_run.js"></script>
     <script type="text/javascript" src="javascripts/background_scripts/posting_process.js"></script>
     <script type="text/javascript" src="javascripts/background_scripts/reading_process.js"></script>
     <script type="text/javascript" src="javascripts/background_scripts/modal_button.js"></script>
+    <script type="text/javascript" src="javascripts/background_scripts/embeded_posting.js"></script>
 
     <!-- test loader to selectively load testing library -->
     <script type="text/javascript" src="javascripts/background_scripts/test_loader.js"></script>
```

Comment:
Oh. are you referring to the test loader?
it is cleared in later versions (for example, in [seamless-posting](https://github.com/SummerWish/privly-chrome/blob/seamless-embeded-posting/background.html)) by this [commit](https://github.com/privly/privly-chrome/commit/43734a3218919070537ca776f412d73fb2670a31)